DQM/CastorMonitor/interface/CastorDigiMonitor.h

```diff
@@ -44,6 +44,12 @@ class CastorDigiMonitor: public CastorBaseMonitor {
   MonitorElement *hSector;
   MonitorElement* hdigisize;
 //  MonitorElement* hBunchOcc;
+
+  int TS_MAX = 10;
```

Comment:
I missed it was set in the constructor. However, it could become ```const``` by setting the value before the body of the constructor.